From 2b351b588a63bff62c22ee916c7b589c056fdb2f Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 11:42:47 +0800
Subject: [PATCH] do not support ISO hold by hard drive partitions

We do not plan to have hard drive partitions to hold an ISO.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 42 on 20250414.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/ui/gui/spokes/installation_source.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/spokes/installation_source.py b/pyanaconda/ui/gui/spokes/installation_source.py
index 939e221a5e..6cbc547f63 100644
--- a/pyanaconda/ui/gui/spokes/installation_source.py
+++ b/pyanaconda/ui/gui/spokes/installation_source.py
@@ -655,7 +655,8 @@ class SourceSpoke(NormalSpoke, GUISpokeInputCheckHandler, SourceSwitchHandler):
 
             added = True
 
-        # Again, only display these widgets if an HDISO source was found.
+        # Do not support an HDISO source.
+        added = False
         self._iso_box.set_no_show_all(not added)
         self._iso_box.set_visible(added)
         self._iso_button.set_no_show_all(not added)
-- 
2.7.4

